### PR TITLE
change Ortto fields to lowercase

### PR DESCRIPTION
**Pull request by RamRamez**

### CODE REVIEW

File: src/services/notificationService.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ const activityCreator = (payload: any, orttoEventName: NOTIFICATIONS_EVENT_NAMES
         "str:cm:tokensymbol": payload.tokenSymbol,
         "str:cm:email": payload.email,
         "str:cm:userId": payload.userId?.toString(),
```
Comment (divine-comedian) [inline src/services/notificationService.ts:39]: can you change this also to lowercase?

File: src/services/notificationService.ts [hunk 2]
```diff
@@ -46,7 +46,7 @@ const activityCreator = (payload: any, orttoEventName: NOTIFICATIONS_EVENT_NAMES
...
```
Comment (divine-comedian) [inline src/services/notificationService.ts:48]: can you change this also to lowercase?

**Review by divine-comedian — CHANGES_REQUESTED** [review]

File: src/services/notificationService.ts (hunk 1 above)
Comment (RamRamez) [reply src/services/notificationService.ts]: Done!

File: src/services/notificationService.ts (hunk 2 above)
Comment (RamRamez) [reply src/services/notificationService.ts]: Done!

**Review by divine-comedian — APPROVED** [review]

**Pull request merged**